modules/custom/dkan_search/src/WebServiceApi.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+namespace Drupal\dkan_search;
+
+use Drupal\dkan_common\JsonResponseTrait;
+use Drupal\dkan_metastore\Service;
+use Drupal\search_api\Query\QueryInterface;
+
+/**
+ * Controller.
+ */
+class WebServiceApi {
+  use JsonResponseTrait;
+
+  /**
+   * Search.
+   */
+  public function search() {
+    $storage = \Drupal::service("entity.manager")->getStorage('search_api_index');
```

Comment:
isn't `entity.manager` deprecated? I believe `entity_type.manager` is recommended.